config/cesm/config_grids.xml

```diff
@@ -1154,7 +1157,7 @@
     </domain>
 
     <domain name="gland4">
-      <nx>376</nx> <ny>701</ny>
+      <nx>416</nx> <ny>704</ny>
```

Comment:
Tony was right to change that. It turns out these values had been wrong for a long time but apparently this didn't cause any trouble outside of an X compset.